go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/googleapis/gax-go/v2 v2.0.5 // indirect
 	github.com/googleapis/gnostic v0.1.0 // indirect
 	github.com/gophercloud/gophercloud v0.0.0-20181215224939-bdd8b1ecd793 // indirect
+	github.com/gorilla/csrf v1.6.0
```

Comment:
Can you please put this change and the vendored code into its own commit? `git reset HEAD~1` will put all these files back into pending status.